WSL/use-custom-distro.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use any Linux distribution with WSL
+description: Learn how to use any Linux distribution within WSL using the import command.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem, distro, custom
+ms.date: 02/17/2021
+ms.topic: article
+---
+
+# Use any Linux distribution with WSL 
+
+You can use any Linux distribution inside of the Windows Subsystem for Linux, even ones that aren't available in the Microsoft store. This article shows how to use CentOS as a distribution inside of WSL. 
+

```

Comment:
You can use any Linux distribution inside of the Windows Subsystem for Linux (WSL), even if it is not available in the [Microsoft Store](https://www.microsoft.com/en-us/search/shop/apps?q=linux) by importing it with a tar file. 

This article shows how to import the Linux distribution, CentOS, for use with WSL by obtaining a tar file using a Docker container.